File: rest/ip-access-control-list/create-ip-acl/create-ip-acl.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "{{ account_sid }}"; 
$token = "{{ auth_token }}"; 
$client = new Services_Twilio($sid, $token);

$ip_access_control_list = $client->account->sip->ip_access_control_lists->create("My new acl", array());
echo $ip_access_control_list->sid;

File: rest/ip-access-control-list/create-ip-acl/create-ip-acl.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = '{{ account_sid }}'
auth_token = '{{ auth_token }}'
@client = Twilio::REST::Client.new account_sid, auth_token

ip_access_control_list = @client.account.sip.ip_access_control_lists.create(:friendly_name => "My new acl")
puts ip_access_control_list.friendly_name
